web/docs/operators/context.md

```diff
@@ -46,6 +47,8 @@ The `context` operator manages [context](../contexts.md) instances.
 - The `inspect` command dumps a specific context's user-provided data, usually
   the context's content.
 
+- The `rename` operator renames a context.
```

Comment:
Can you explain what `--force` does here?

Reply:
Added an explanation.